baseband/vlbi_base/base.py

```diff
@@ -49,27 +48,19 @@ class VLBIStreamBase(VLBIFileBase):
 
     _frame_class = None
     _squeezed_shape = None
+    _samples_per_frame = None
```

Comment:
I don't think we need to define these since you set them unconditionally in `__init__`. (Having them here suggests it might be relevant for subclasses to override them, but that is not the case).